downstream/modules/builder/proc-installing-builder.adoc

```diff
@@ -2,12 +2,22 @@
 
 = Installing {Builder}
 
-You can install {Builder} using Red Hat Subscription Management to register and attach to your {PlatformName} subscription. In your terminal, run the following command:
+You can install {Builder} using Red Hat Subscription Management to register and attach to your {PlatformName} subscription.
+
+.Prerequisites
+* You have access to a valid {PlatformName} subscription.
+* You chose and downloaded the ansible-builder package from https://access.redhat.com/downloads/content/rhel---8/x86_64/11844/ansible-builder/1.0.1-2.el8ap/noarch/fd431d51/package[Red Hat Ansible Automation Platform Packages].
+
+.Procedure
+In your terminal, run the following command:
 ----
 $ dnf install ansible-builder
 ----
 
 You can also install {Builder} from the https://pypi.org/project/ansible-builder/[Python Package Index (PyPI)]. To install with this setup, run:
+
+NOTE: If you encounter issues while installing from PyPI, please install and reproduce on a Supported installation.
```

Comment:
Should the note follow the `pip install` command?

Reply:
You're right @ariordan-redhat. I fixed it.